application/utils/noise_filter/pipeline.py

```
@@ -0,0 +1,140 @@
+        is_noise, _reason = regex.is_noise_record(record)
+        if is_noise:
+            summary.dropped_noise += 1
+        else:
+            survivors.append(_sanitized(record))
+
+    # Stage 2: LLM classify.
+    classifier = classifier or LLMClassifier(config)
+    verdicts = classifier.classify_batch(survivors)
+
+    triples = [
+        (rec, v, compute_content_hash(rec.text)) for rec, v in zip(survivors, verdicts)
```

Ruff (0.16.0)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)